apps/prairielearn/elements/pl-integer-input/pl-integer-input.py

```diff
@@ -321,14 +321,6 @@ def test(element_html: str, data: pl.ElementTestData) -> None:
     if a_tru_parsed is None:
         raise ValueError(f"Could not parse correct answer: {a_tru}")
 
-    # TODO: Enable this typeguard
-    # if not isinstance(
-    #     a_tru_parsed, int | float | complex | np.complexfloating | np.number
-    # ):
-    #     raise TypeError(
-    #         f"Correct answer '{a_tru}' is not a valid type, got {type(a_tru)}"
-    #     )
-
     result = data["test_type"]
```

Comment:
why getting rid of this?

Reply:
This was a remnant of `from_json` having a poor type annotation. I decided that improving the typing of `from_json` should be a separate PR, as it is pretty hard to do  correctly.